tests/contrib/molten/test_molten.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import molten
+from ddtrace import Pin
+from ddtrace.contrib.molten import patch
+from molten import App, Route
+from molten.testing import TestClient
+from nose.tools import eq_, ok_
```

Comment:
Please do not use `nose`, we want to get rid of it :)

You can use `self.assertEqual()`, and `self.assertTrue()` (or any of the other built in `unittest.TestCase` assertion methods).